server/index.js

```diff
@@ -414,6 +415,15 @@ app.prepare().then(() => {
       });
     }
   });
+  server.get('/team-list', async (req, res) => {
+    const team = await getTeam();
+    res.send(JSON.stringify(team));
+  });
+
+  server.get('/careers-list', async (req, res) => {
+    const careers = await getCareers();
+    res.send(JSON.stringify(careers));
+  });
```

Comment:
Please rename routes to be `/astronauts` and `/careers`.

Reply:
I have left `careers` because when I set it I got broken page

![image](https://user-images.githubusercontent.com/1932290/55875221-304f3080-5b9d-11e9-9041-268ab74b8c21.png)

Reply:
I have left careers because when I set it I got broken page

![image](https://user-images.githubusercontent.com/37536815/55875244-46f58780-5b9d-11e9-98c1-6fce927e7614.png)

Reply:
Could it be `/api/careers` and `/api/team-members`?